src/NServiceBus.Core/Transports/Msmq/MsmqTransport.cs

```diff
@@ -33,9 +33,13 @@ public override TransportInfrastructure Initialize(SettingsHolder settings, stri
             var msmqSettings = connectionString != null ? new MsmqConnectionStringBuilder(connectionString)
                 .RetrieveSettings() : new MsmqSettings();
 
+            msmqSettings.UseDeadLetterQueueForMessagesWithTimeToReachQueue = settings.GetOrDefault<bool>(UseDeadLetterQueueForMessagesWithTimeToReachQueue);
```

Comment:
I tried that but we have no good place to set the defaults so it just meant one extra line in the same method so I decided to keep as is